docs/usage/validators.md

```diff
@@ -321,13 +345,18 @@ print(Model(a='invalid').a)
 #> 2000-01-01 00:00:00
 ```
 
-## Validate always
+### Before and after validators
+
+In addition, you can execute code *before* or *after* field validation
+with the help of the `@field_validator` decorator `mode` parameter.
 
-**TODO this content is wrong!**
+`@field_validator`s can have `mode='before'`, in which case they're called before regular
+validation occurs (and are provided with the raw input data), or `mode='after'` (the default), in which case
+they're called after validation.
```

Comment:
Worth mentioning the other modes, `"wrap", "plain"` and how they can be applied in annotated validators, but not decorated validators?

Reply:
Adding a note and link to Annotated validators